shared/css/color.scss

```diff
@@ -118,5 +118,47 @@ $droplet_purple: #bb77c7;
 $droplet_green: #68d995;
 $droplet_white: $white;
 
+// Applab theme values.
```

Comment:
I'm not sure it's super useful for these to be in the global color.scss unless you think they'll get used outside of App Lab.

Reply:
Some of the default applab values were there, so I tried to keep them in the same place. I agree that we should look at moving these out.